tests/dummy/app/index.html

```diff
@@ -29,8 +29,14 @@
 
     <script src="{{rootURL}}assets/vendor.js"></script>
     <script src="{{rootURL}}assets/dummy.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js" integrity="sha384-0mSbJDEHialfmuBBQP6A4Qrprq5OVfW37PRR3j5ELqxss1yVqOtnepnHVP9aJ7xS" crossorigin="anonymous"></script>
 
+    <!-- load jquery, purely to resolve the bootstrap nag -->
```

Comment:
I think we can get rid of the:

<img width="547" alt="image" src="https://user-images.githubusercontent.com/662944/44132181-a2f2b6d6-a00c-11e8-8153-a9c5ee8ff350.png">



 nag by bumping to bootstrap 3.3.7 instead of adding a second copy of jquery: http://blog.getbootstrap.com/2016/07/25/bootstrap-3-3-7-released/

Reply:
good call.